servatrice/migrations/servatrice_0009_to_0010.sql

```diff
@@ -0,0 +1,12 @@
+-- Servatrice db migration from version 9 to version 10
+
+CREATE TABLE IF NOT EXISTS `cockatrice_warnings` (
+  `user_name` varchar(255) NOT NULL,
```

Comment:
These should be by id instead of by name, I'd think, since it's an FK to the users table